content/operate/rc/security/access-control/saml-sso/saml-integration-okta-generic.md

```diff
@@ -256,7 +253,7 @@ To activate SAML, you must have a local user (or social sign-on user) with the *
 
     {{<image filename="images/rc/saml/sm_saml_8.png" >}}
 
-  A popup appears, stating that to test the SAML connection, you need to log in with Okta credentials of the user defined in the Redis Cloud group. This user is part of the group to which you assigned the Redis Cloud application.
+    A popup appears, stating that to test the SAML connection, you need to log in with Okta credentials of the user defined in the Redis Cloud group. This user is part of the group to which you assigned the Redis Cloud application.
```

Comment:
Bit awkward, maybe? You could perhaps reorder it a bit, along the lines of "The app displays a popup to inform you that you must log in with the Okta credentials of the user you defined in the Redis cloud group if you want to test the SAML connection."